frontend/src/lib/search-utils.ts

```diff
@@ -0,0 +1,5 @@
+/* Copyright Contributors to the Open Cluster Management project */
+
+export const handleOperatorComparison = (value: string, selectedValue: string) => {
```

Comment:
Based on the name, I guess this will be updated later to also accept an operator.

Reply:
Yes, it had an operator param and switch in my first draft, but I removed it. Serves no purpose with only one supported operator. This is expanded in iteration 2.